### ISSUE TITLE
Exception when trying to save zmd: System.NullReferenceException

### ISSUE THREAD

**Issue by Ben1138** [marth8880/ZeroMunge#16]
When list of Actions contains an empty entry, NullReferenceException occurs while saving:
```
   at ZeroMunge.ZeroMunge.SerializeData(String filePath)
   at ZeroMunge.ZeroMunge.SaveFileListToFile(String filePath)
   at ZeroMunge.ZeroMunge.saveDlg_SaveFileListPrompt_FileOk(Object sender, CancelEventArgs e)
   at System.Windows.Forms.FileDialog.OnFileOk(CancelEventArgs e)
   at System.Windows.Forms.FileDialog.HandleVistaFileOk(IFileDialog dialog)
```

1. Open ZeroMunge
2. In the Action List, set checkmark at Process (will cerate new Action)
3. Click File -> Save As -> somefile.zmd

**Comment by marth8880** [comment]
Could you add a repro for this? Thanks!

**Comment by marth8880** [comment]
Issue fixed in 4259114 - closing.
